FlorenceBML/BMLacq756/BMLacq756.xml

```diff
@@ -29,7 +29,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <ref type="work" corresp="LIT6845PPSatolay"/>.</note>
+                            <incipit xml:lang="gez">ጸሎት፡ በእንተ፡ ሸቶላይ፡ ወሸቶላዊት፡ አጋንንት፡ ዘትቀትሎ፡ ሕፃናተ፡ ወታወሕዙ፡ ደመ፡ አመሐኩክሙ፡ ወአውገዝኩከሙ፡ ወአርሰርኩክሙ፡</incipit>
+                            <explicit xml:lang="gez">ደፈር፡ በሎንቃር፡ ፈውስ፡ ወአሕይዋ፡ ለአመትከ፡ 
+                                <hi rend="rubric"><persName ref="PRS14227WalattaSellase">ወልተ፡ ሥላሴ፡</persName></hi>እመ፡ አልቦ፡ ነገር፡ ዘይሰአኖ፡ 
+                                ለእግአብሔር፡</explicit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support><material key="parchment"/></support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <note>Composed of three parts (670+460+410 mm) that are sewn together with thin leather strips.</note>
+                                    <dimensions unit="mm">
+                                        <height>1540</height>
+                                        <width>98</width>                                      
+                                    </dimensions>
+                                </extent>
+                                <condition key="good"/>
+                            </supportDesc>
+                            <layoutDesc><layout columns="2" writtenLines="180 188">
+                                <note>181 lines in the left column and 188 lines in the right column.</note>
+                                <ab type="pricking">No pricking/></ab>
+                                <ab type="ruling">No ruling</ab>
+                            </layout>
+                                <layout columns="1" writtenLines="2">
+                                    <ab type="pricking">No pricking/></ab>
+                                    <ab type="ruling">No ruling</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc><handNote xml:id="h1" script="Ethiopic">
+                            <ab type="script"/>
+                            <seg type="script">minute and accurate</seg>
+                            <seg type="ink">Black, red</seg>
+                            <seg type="rubrication">incipits, sacred names, name of owner</seg>
+                            <date from="1700" to="1900" cert="low" datingMethod="paleography"/>
+                          </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">Large miniature in black and red, depicting an angel with a sword, surrounded 
```

Comment:
I believe `<desc>` is necessary inside decoNote

Reply:
Done.